wallet/dapps-communication.md

```diff
@@ -1,11 +1,366 @@
-# Wallet-DApp communication standard
+# Wallet <> DApp communication
 
 Initial meta issue: <https://github.com/massalabs/massa-standards/issues/13>
 
+**Authors:**
+
+**Status:** Draft
+
+**Version:** 0.1
+
 ## Abstract
 
-## Motivation
+This standard defines a protocol for communication between a cryptocurrency wallet and a decentralized application (DApp) running in a web browser. The protocol provides a standard interface for DApps to request user authorization for blockchain transactions, as well as a standard mechanism for wallets to sign and broadcast those transactions. By following this standard, DApp developers can provide a seamless user experience for users of any wallet that implements the protocol, while wallet developers can ensure that their products are compatible with a wide range of DApps.
+
+## Targeted Audience
+This standard is targeted towards developers who are building decentralized applications (DApps) and cryptocurrency wallets that need to interact with each other in a browser environment. 
+
+The standard assumes a working knowledge of web development and blockchain technology. It is also assumed that the reader has experience with browser extensions and understands the basic principles of secure communication between web pages and extensions.
+
+> _NOTE:_ For more information on [content scripts](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Content_scripts) and [background scripts](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Background_scripts), see [MDN's browser extensions pages](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions).
 
 ## Specification
 
+### Event-Based Communication
+Communication between the DApp and the wallet will use event-based messaging. There are two types of events: 
```

Comment:
we have a misunderstanding here... let's discuss about it ASAP

Reply:
it's okay in fact, the example implementation bellow follow what is explain here.